app/content/overview.md

```diff
@@ -68,6 +68,14 @@ PUBLICATION_ID=publisher-center-ppid.google.com
 OAUTH_CLIENT_ID=abcd-1234.apps.googleusercontent.com
 OAUTH_CLIENT_SECRET=secret-abc-1234
 GOOGLE_SITE_VERIFICATION=public-abc-1234
+
+# Google Analytics
+GTAG_PROPERTY_ID: G-1234567890  #Configure gTag Id
```

Comment:
mega-nit, space after # before comment e.g. "# Configure gTag Id"

Reply:
@Wayne-X oh actually the `:` is a typo. Will push a fix...